db/seeds/development/01_departments.rb

```diff
@@ -0,0 +1,16 @@
+require Rails.root.join('db', 'seeds', 'support', 'department_seeder')
+
+seeder = DepartmentSeeder.new
+
+departments =  ["/dev/one",
+                "/dev/two",
+                "/dev/tre",
+                "/dev/ruby",
+                "/mid",
+                "/ux",
+                "/zh",
+                "/sys",
+                "/bs",
+                "Funktionsbereiche"]
+
+seeder.seed_departments(departments)
```

Comment:
I think you could remove this extra seeder class. It doesn't do a lot.
department_names.each {|n| Department.seed(name: n) }